projects/core/src/lib/datasets/upload/upload.component.ts

```diff
@@ -84,9 +94,24 @@ export class UploadComponent {
             spatialReference: new UntypedFormControl('EPSG:4326', Validators.required), // TODO: validate sref string
         });
 
-        this.formNameDescription = new UntypedFormGroup({
-            name: new UntypedFormControl('', Validators.required),
-            description: new UntypedFormControl(''),
+        this.formNameDescription = new FormGroup<NameDescription>({
+            name: new FormControl('', {
+                nonNullable: true,
+                validators: [Validators.required, Validators.pattern(/^[a-zA-Z0-9_]+$/), Validators.minLength(1)],
```

Comment:
I get this error when using asdf as name and wertz as display name: 

> {"error":"BodyDeserializeError","message":"invalid character '?' in named data at line 1 column 105"}


idk why it uses "?" as namespace

Reply:
Buggy for non-pro
[b7cba87](https://github.com/geo-engine/geoengine-ui/pull/624/commits/b7cba87e7766e54bb38698eaa1f4314cbcfedbbc)

Reply:
![image](https://github.com/geo-engine/geoengine-ui/assets/10283892/1f0a0e57-d756-4487-8041-c97d4eaf0421)
does it really make sense to show the namespace here? You can't change it anyway, and you also don't really need to know it right? Doesn't look that nice.

Reply:
Well, it's the name you will access it with.